torch/_dynamo/variables/builtin.py

```diff
@@ -278,6 +289,13 @@ def call_function(
                         need_unwrap=need_unwrap,
                         **options,
                     )
+                elif all(isinstance(x, DynamicShapeVariable) for x in args):
```

Comment:
This doesn't seem very robust as if one of the args is a ConstantVariable it will fall through to the next case.

Perhaps we could inspect the type of `proxy` instead.  I am assuming in this case it won't actually be an FX proxy.